STYLEGUIDE.adoc

```diff
@@ -110,6 +126,14 @@ By default, all of these use the class name as label, but that can be customized
 javadoc:hudson.scm.SCM#all()[a list of all known SCM implementations]
 ----
 
+== Handbook Style Guide
+
+* With sections that are related/build onto each other. THere should be a
```

Comment:
For consecutive sections that are related to or build on each other, there should